lua/null-ls/builtins/diagnostics/cspell.lua

```diff
@@ -14,11 +32,23 @@ return h.make_builtin({
     generator_opts = {
         command = "cspell",
         args = function(params)
-            return {
+            local cspell_args = {
+                "lint",
                 "--language-id",
                 params.ft,
                 "stdin",
             }
+
+            local should_add_suggestions = require("null-ls").get_source({
+                name = "cspell",
+                method = methods.internal.CODE_ACTION,
+            }) ~= nil
```

Comment:
```suggestion
            local should_add_suggestions = not vim.tbl_isempty(require("null-ls").get_source({
                name = "cspell",
                method = methods.internal.CODE_ACTION,
            }))
```

I am not good at naming API methods (and I need to think about changing the name completely, since this mistake is too easy to make).

Reply:
Rebased the change into the right commit to keep the history clean.